x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -431,6 +431,8 @@ export const AddExceptionFlyoutWrapper: React.FC<AddExceptionFlyoutWrapperProps>
     enrichedAlert == null ||
     (memoRuleIndices == null && memoDataViewId == null);
 
+  if (isLoading || isRuleLoading) return null;
```

Comment:
wonder if it possible to display here any kind of loader? So if it takes longer time to load users might think nothing is happening

Reply:
Ideally, the loader should be in the flyout, but we hide the flyout to fix this bug.